From 105e6b1e6820832ffa99be50bc3f8738a3cf2a07 Mon Sep 17 00:00:00 2001
From: "chen.pj" <chen.pj@inventec.corp-partner.google.com>
Date: Wed, 3 May 2023 05:37:03 +0000
Subject: [PATCH] [aspeed][mctp] - Fix peci-mctp device name error

Symptom/Reason:
    Cannot create device peci-mctp.

Root Cause:
    Aspeed's patch change the peci-mctp device name,
    and not match the peci-mctp driver name set by Intel.

Solution/Change:
    Change the platform device name to match setting.

Entry Test:

sysadmin@superion:~# cat /sys/bus/peci/devices/peci-1/name
peci-mctp
---
 drivers/soc/aspeed/aspeed-mctp.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/soc/aspeed/aspeed-mctp.c b/drivers/soc/aspeed/aspeed-mctp.c
index d1c4b94951ee..d568c9e06607 100644
--- a/drivers/soc/aspeed/aspeed-mctp.c
+++ b/drivers/soc/aspeed/aspeed-mctp.c
@@ -2198,7 +2198,7 @@ static int aspeed_mctp_probe(struct platform_device *pdev)
 	}
 
 	priv->peci_mctp = platform_device_register_data(
-		priv->dev, priv->rc_f ? "peci1-mctp" : "peci0-mctp",
+		priv->dev, priv->rc_f ? "peci1-mctp" : "peci-mctp",
 		PLATFORM_DEVID_NONE, NULL, 0);
 	if (IS_ERR(priv->peci_mctp))
 		dev_err(priv->dev, "Failed to register peci-mctp device\n");
-- 
2.17.1

